examples/interactive_play.py

```diff
@@ -98,10 +103,15 @@ def get_input_vel_ctlr(
     else:
         agent_k = ""
     arm_action_name = f"{agent_k}arm_action"
-    base_action_name = f"{agent_k}base_velocity"
-    arm_key = "arm_action"
-    grip_key = "grip_action"
-    base_key = "base_vel"
+
+    if use_joint_control:
+        base_action_name = f"{agent_k}humanjoint_action"
```

Comment:
what happens if `use_joint_control=True` but there is no human in the task ?

Reply:
Same comment as above.